src/components/PreviousTasks/PreviousTasksCount.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import PreviousTasksCount from './PreviousTasksCount';
+
+describe('PreviousTaskCounts', () => {
+  it('should not render if number of previous tasks is zero', () => {
+    const { container } = render(
+      <PreviousTasksCount numberOfPreviousTasks={0} />,
+    );
+    const previousTaskContainer = container.getElementsByClassName('previous-task-container');
+    expect(previousTaskContainer).toHaveLength(0);
+  });
+
+  it('should render with singular text if number of previous tasks is one', () => {
+    const { container } = render(
+      <PreviousTasksCount numberOfPreviousTasks={1} />,
+    );
+    const previousTaskContainer = container.getElementsByClassName('govuk-!-margin-left-6');
+    expect(previousTaskContainer[0].textContent).toContain('1 previous task');
+  });
+
+  it('should  render with plural text if number of previous tasks is greater than one', () => {
+    const { container } = render(
+      <PreviousTasksCount numberOfPreviousTasks={2} />,
+    );
+    const previousTaskContainer = container.getElementsByClassName('govuk-!-margin-left-6');
```

Comment:
```suggestion
    const previousTaskContainer = container.getElementsByClassName('previous-task-container');
```

The `.textContent` should also work too with the suggested change above